ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/tools/BehaviorStoredPropertySetDefinition.java

```diff
@@ -19,58 +20,91 @@
 
 public class BehaviorStoredPropertySetDefinition extends CRDTBidirectionalStoredPropertySet
 {
+   private final String name;
+   private final TObjectDoubleHashMap<DoubleStoredPropertyKey> defaultDoubleValues = new TObjectDoubleHashMap<>();
+   private final TObjectIntHashMap<IntegerStoredPropertyKey> defaultIntegerValues = new TObjectIntHashMap<>();
+   private final TObjectByteHashMap<BooleanStoredPropertyKey> defaultBooleanValues = new TObjectByteHashMap<>();
    private final TDoubleArrayList onDiskDoubleValues = new TDoubleArrayList();
    private final TIntArrayList onDiskIntegerValues = new TIntArrayList();
    private final TByteArrayList onDiskBooleanValues = new TByteArrayList();
 
    public BehaviorStoredPropertySetDefinition(RequestConfirmFreezable requestConfirmFreezable,
+                                              String name,
                                               StoredPropertySetBasics storedPropertySet)
    {
       super(requestConfirmFreezable, storedPropertySet);
 
+      this.name = name;
+
+      storedPropertySet.getKeyList().keys().forEach(key ->
+      {
+         if (key instanceof DoubleStoredPropertyKey doubleKey)
+         {
+            defaultDoubleValues.put(doubleKey, storedPropertySet.get(doubleKey));
+         }
+         else if (key instanceof IntegerStoredPropertyKey integerKey)
+         {
+            defaultIntegerValues.put(integerKey, storedPropertySet.get(integerKey));
+         }
+         else if (key instanceof BooleanStoredPropertyKey booleanKey)
+         {
+            defaultBooleanValues.put(booleanKey, booleanToByte(storedPropertySet.get(booleanKey)));
+         }
+      });
+
       setOnDiskFields();
    }
 
    public void toJSON(ObjectNode jsonNode)
    {
       StoredPropertySetReadOnly storedPropertySet = getValueReadOnly();
-      ObjectNode objectNode = jsonNode.putObject(storedPropertySet.getTitle());
+      ObjectNode objectNode = jsonNode.putObject(name);
 
       for (StoredPropertyKey<?> key : storedPropertySet.getKeyList().keys())
       {
-         if (key instanceof DoubleStoredPropertyKey doubleKey)
+         if (key instanceof DoubleStoredPropertyKey doubleKey
+             && !(defaultDoubleValues.get(doubleKey) == storedPropertySet.get(doubleKey)))
          {
-            objectNode.put(doubleKey.getTitleCasedName(), storedPropertySet.get(doubleKey));
+            objectNode.put(doubleKey.getTitleCasedName(), "%s -> %s".formatted(defaultDoubleValues.get(doubleKey),
+                                                                               storedPropertySet.get(doubleKey)));
```

Comment:
Showing what the default is plus the current value is really useful in understanding the intent.